python/neuroglancer/pipeline/scripts/compute_bounds.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import os
```

Comment:
One warning, it relies on removing a bug/design flaw in Storage.list_files. That will require another PR.